.github/workflows/deploy.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       - name: Check out repository
         uses: actions/checkout@v2
         with:
-          fetch-depth: 10
+          fetch-depth: 20
```

Comment:
what’s up with fetch-depth?

Reply:
ohhh, yeah so when I was hitting failure after failure with each commit in my branch I hit an issue with the CI skipping script where lerna wouldn't be able to find the commit in it's history, because the GHA only fetches the past 10 commits. 

Anyway, this was to give more git history to GHA so lerna could do it's thing. I have a bug ticket open to clean up the skipping script to deal with this a bit better.